discoverers/discoverer_aws_ec2.go

```diff
@@ -390,7 +410,16 @@ func (ec2d *AwsEc2Discoverer) reachabilityCheck(
 	ports := ec2d.networkReachabilityCheckPorts.Slice()
 	for _, ip := range ips {
 		for _, port := range ports {
-			if openPort(ctx, ip, port) {
+			address := fmt.Sprintf("%s:%s", ip, port)
```

Comment:
code nit: it's a common pattern to create a function for CacheKey, something like that,